static/js/notes.js

```diff
@@ -52,4 +137,26 @@ $(function() {
             });
         });
     });
+
+    $('#add-note-button').click(function() {
+        activateModal($addNoteModal);
+    });
+
+    $(document).on('click', classesByName.noteTypeButton, function() {
```

Comment:
is this to allow you to get radio button like features. Where in clicking one note type it de-selects any other notetype that was selected? 

And we created a new one from scratch because radio buttons don't allow you to deselect/ leave it blank?

Reply:
I'm totally cool with this. Just to know on the side, are radio buttons bad on mobile?

Reply:
uh i'm honestly not sure. @riceballthief do you know?

Reply:
Radio buttons are only use when there are two ore more mutually exclusive options and the users must select only one choice. It's usually good practice to use them to change settings but  never as action buttons. Also, in terms of visual design, it's usually best to arrange items vertically in a logical order. It's not bad to use on mobile in general, there're just appropriate and inappropriate situations to use them, and this goes for non-mobile as well.